lab-examples/clos02/setup.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
you may want to step this game a bit and launch configuration steps concurrently, checking that they all succeed in the end

for example like here https://stackoverflow.com/a/29535256/4902910

Reply:
Added.